### PR TITLE
Added stability to OSS instrumentation tests

### PR DESCRIPTION
**Pull request by bestander**
This change makes all instrumentation tests to be executed in sequence in independent retriable processes.

With a new test being open sourced recently our CI stability degraded.
This PR should bring back stability because tests won't affect each other and will have shorter lifetime

### CODE REVIEW

**Comment by facebook-github-bot-7** [comment]
By analyzing the blame information on this pull request, we identified @mkonicek to be a potential reviewer.

**Comment by facebook-github-bot-8** [comment]
@bestander updated the pull request.

**Comment by facebook-github-bot-1** [comment]
@bestander updated the pull request.

**Comment by facebook-github-bot-4** [comment]
@bestander updated the pull request.

**Comment by facebook-github-bot-1** [comment]
@bestander updated the pull request.

**Comment by facebook-github-bot-4** [comment]
@bestander updated the pull request.

**Comment by facebook-github-bot-0** [comment]
@bestander updated the pull request.

**Comment by facebook-github-bot-1** [comment]
@bestander updated the pull request.

**Comment by facebook-github-bot-4** [comment]
@bestander updated the pull request.

**Comment by facebook-github-bot-9** [comment]
@bestander updated the pull request.

**Comment by facebook-github-bot-2** [comment]
@bestander updated the pull request.

File: scripts/run-android-instrumentation-tests.sh [hunk 1]
```diff
@@ -1,20 +1,26 @@
...
 import re
 import subprocess as sp
 import sys
 import threading
 import time
 
 done = False
+
 test_app = sys.argv[1]
+test_class = sys.argv[2]
```
Comment (mkonicek) [inline scripts/run-android-instrumentation-tests.sh]: Idea: make this optional, that way it's still possible to not pass this argument an execute all tests locally (for easy local testing of the whole suite).

**Comment by mkonicek** [comment]
Cool idea! Should make the runs much faster once there are many tests.

**Comment by facebook-github-bot-5** [comment]
@bestander updated the pull request.

**Comment by bestander** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-0** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/1250749274952341/int_phab) to review.

**Pull request closed without merging**